pkg/ddevapp/ddevapp_test.go

```diff
@@ -2313,7 +2361,7 @@ func readLastLine(fileName string) (string, error) {
 // TestDdevFullSiteSetup tests a full import-db and import-files and then looks to see if
 // we have a spot-test success hit on a URL
 func TestDdevFullSiteSetup(t *testing.T) {
-	if runtime.GOOS == "windows" || dockerutil.IsColima() || dockerutil.IsLima() || dockerutil.IsRancherDesktop() {
+	if os.Getenv("DDEV_RUN_TEST_ANYWAY") != "true" && (runtime.GOOS == "windows" || dockerutil.IsColima() || dockerutil.IsLima() || dockerutil.IsRancherDesktop()) {
```

Comment:
Has `DDEV_RUN_TEST_ANYWAY` been introduced for any future use?
I don't see how it can be used, as I usually run one test locally. And I don't see any usage in GitHub Actions.

Reply:
This is the first time I used it. I was testing using Lima, and it was really annoying. So it's intended here only for local test running. In the past when I was testing something like this I would comment out this line or equivalent line, but then it's likely to get accidentally committed.